shell/platform/common/cpp/public/flutter_texture_registrar.h

```diff
@@ -0,0 +1,62 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
Note: you'll need to update flutter/flutter [in several places](https://github.com/flutter/flutter/search?q=flutter_plugin_registrar.h&unscoped_q=flutter_plugin_registrar.h) once this lands.